modules/consul-cluster/variables.tf

```diff
@@ -184,3 +184,9 @@ variable "ssh_port" {
   description = "The port used for SSH connections"
   default     = 22
 }
+
+variable "tags" {
+  description = "List fo extra tag blocks added to the autoscaling group configuration"
```

Comment:
Please mention that each block needs to specify `key`, `value`, and `propagate_at_launch`.

Reply:
Will do